content/docs/reference-javascript-environment-requirements.md

```diff
@@ -23,8 +23,8 @@ ReactDOM.render(
 );
 ```
 
-React also depends on `requestAnimationFrame` (even in test environments).  
-You can use the [raf](https://www.npmjs.com/package/raf) package to shim `requestAnimationFrame`:
+React 也依賴於 `requestAnimationFrame`（甚至包括測驗環境）。
+你可以使用[raf](https://www.npmjs.com/package/raf) package 去 shim `requestAnimationFrame`：
```

Comment:
```suggestion
你可以使用 [raf](https://www.npmjs.com/package/raf) package 去模擬 `requestAnimationFrame`：
```

The term `shim` not sure can translate `模擬`.

Reply:
How about `shim` translate? Or maybe keep original term? cc @chloewlin @itszero

Reply:
I would vote for original because I really can't think of a good translation. If we do want to translate, I think a functional translation would be easier: something like "提供" as in providing that function.

Reply:
I think 模擬

Reply:
I agree to keep the original, thank you for feedback.

Reply:
agree, thanks